gradle.properties

```diff
@@ -12,7 +12,7 @@ archives_base_name=miningutility
 # Dependencies
 # check this on https://modmuss50.me/fabric.html
 fabric_version=0.89.0+1.20.1
-yacl_version=3.2.1+1.20
+midnightlib_version=1.4.1-fabric
```

Comment:
Please revert all config related changes - I'd rather use only YACL and not rely on 3rd party libraries for the core config functionality.